data-raw/01_gut_taxonomy_annotate_df.R

```diff
@@ -0,0 +1,2 @@
+gut_taxonomy_annotate_df <- sourmashconsumr::read_taxonomy_annotate(Sys.glob("inst/extdata/SRR*with-lineages*csv.gz"))
+usethis::use_data(gut_taxonomy_annotate_df, overwrite = TRUE, compress = "xz")
```

Comment:
why are the `gut_gather_df`, `gut_signatures_df` and `gut_taxonomy_annotate_df` called in separate R script in the `data-raw` foler?

Reply:
🤷‍♀️ it was just the convention documented in the R package instruction to have one R script per data set that's created

Reply:
interesting...